src/explore-education-statistics-admin/src/components/comments/Comment.tsx

```diff
@@ -1,47 +1,58 @@
 import styles from '@admin/components/comments/Comment.module.scss';
 import CommentEditForm from '@admin/components/comments/CommentEditForm';
 import { useCommentsContext } from '@admin/contexts/CommentsContext';
-import { Comment as CommentType } from '@admin/services/types/content';
+import {
+  Comment as CommentType,
+  CommentType as Type,
```

Comment:
Maybe to make this aliasing less weird, we could do `Comment as CommentData`. This frees up `CommentType` so we don't need to alias it as `Type`